lib/helpers/constants.dart

```diff
@@ -23,6 +23,7 @@ class RouteConstant {
   static const String TERMS_CONDITIONS = '/terms-conditions';
   static const String PAYMENT_SUCCESSFULL = '/payment-successful';
   static const String CHANGE_USERNAME_SCREEN = '/change_username';
+  static const String UNKNOWN_VIEW = '/unknown-view';
```

Comment:
Do we need this here? I don't see its use case.

Reply:
Fixed!